google/cloud/security/inventory/pipelines/load_firewall_rules_pipeline.py

```diff
@@ -30,14 +30,14 @@ class LoadFirewallRulesPipeline(base_pipeline.BasePipeline):
 
     RESOURCE_NAME = 'firewall_rules'
 
-    def _transform(self, firewall_rules_map):
+    def _transform(self, resource_from_api):
```

Comment:
nit: Is it necessary have "resource_from_api" here?  I realize it's from the base pipeline, but that was a generic placeholder there, because that is an abstract class.  Here in the concrete classes, it's nicer if parameter can reflect the actual resource that is passed in.